isce2_topsapp/delivery_prep.py

```diff
@@ -152,18 +152,21 @@ def format_metadata(nc_path: Path,
     metadata.update({"ogr_bbox": ogr_bbox,
                      "reference_scenes": all_metadata['reference_scenes'],
                      "secondary_scenes": all_metadata['secondary_scenes'],
-                     "sensing_start": startTime_f,
-                     "sensing_stop": stopTime_f,
-                     "orbit_number": [int(ref_props['orbit']),
-                                      int(sec_props['orbit'])],
-                     "platform": [ref_props['platform'], sec_props['platform']],
-                     "beam_mode": ref_props['beamModeType'],
-                     "orbit_direction": ref_props['flightDirection'].lower(),
+                     "sensing_start": ref_start_time_formatted,
+                     "sensing_stop": ref_stop_time_formatted,
+                     "frame_number": all_metadata['frame_id'],
```

Comment:
In cases where frames are not used, I think the ingest code will be more straightforward if the `frame_number` isn't included at all, rather than having `frame_number=1`, e.g.:

```
metadata.update({"ogr_bbox": ogr_bbox,
                 "reference_scenes": all_metadata['reference_scenes'],
                 "secondary_scenes": all_metadata['secondary_scenes'],
                 ...
})

if all_metadata['frame_id'] != -1:
    metadata['frame_number'] = all_metadata['frame_id']

That will let us use the same approach for v2.0.6 scenes and v3.0.0 scenes without a frame number; both will end up in CMR without a FRAME_NUMBER attribute.